### DISCUSSION TITLE
Python (Pyodide) sometimes fails to work

### DISCUSSION THREAD

**Discussion by eugene-f02** [jupyterlite/jupyterlite#863] [Q&A]
Hi,
I'm experiencing a bug with Python (Pyodide). It happens more often when I launch a notebook that is not located inside the root folder.

Here is the error that I'm seeing in the browser's console:

<img width="1432" alt="Screen Shot 2022-11-04 at 1 25 17 PM" src="https://user-images.githubusercontent.com/57968389/200038078-edf2813a-610a-41ee-a249-647d97ffc609.png">

This error causes the notebook to not be able to execute any of the cells.

I was using jupyterlite cli command `jupyter lite serve` to build and serve the jupyterlite instance.

Any help would be really appreciated

**Comment by jtpio** [comment]
Thanks @eugene-f02 for reporting.

Does this happen to you when using a hosted instance of JupyterLite, for example https://jupyterlite.rtfd.io/en/latest/try/lab? Or only when served locally with `jupyter lite serve`?
